libs/canvas.py

```diff
@@ -395,8 +395,6 @@ def bounded_move_vertex(self, pos):
 
         left_index = (index + 1) % 4
         right_index = (index + 3) % 4
-        left_shift = None
```

Comment:
Rationale: Variables are overwritten in all following paths, so they can be removed